doc/user_guide/docker/managing_user_data.md

```diff
@@ -40,80 +32,35 @@ VOLUME=my-vol
 DIR=~/tmp/backup
 ```
 
-## Creating a Volume
-
-The following command creates an empty Docker volume with the name specified by environment variable `$VOLUME`.
-
-```shell
-docker volume create $VOLUME
-```
-
-## Using Your Volume in the AI-Lab Docker Container
-
-See also [Restore volume from a backup](https://docs.docker.com/storage/volumes/#restore-volume-from-a-backup) in official Docker documentation.
-
-The following shell command runs AI-Lab as a Docker container while mounting your volume to folder `/root/notebooks`.
-
-```shell
-docker run \
-  --detach \
-  --publish ${LISTEN_IP}:${PORT}:8888 \
-  --volume $VOLUME:/root/notebooks \
-  exasol/data-science-sandbox:${VERSION}
-```
-
-## Removing Your Volume
-
-If you don't need the contents of your Docker volume anymore then you can use the following command to discard it:
-
-```shell
-docker volume rm ${VOLUME}
-```
-
 ## Creating a Backup of Your Volume
 
 See also [Back up a volume](https://docs.docker.com/storage/volumes/#back-up-a-volume) in official Docker documentation.
 
-The following shell commands
-* Run a Docker command to backup the volume
-  * Option `-rm` removes the container after exit.
-  * The `--volume` options mount your volume and your backup directory to enable the Docker container to access them.
-* The `tar` command executed inside the Docker container uses
-  * Option `-C` to change to directory `/volume`
-  * Option `-c` to create a file archive
-  * Option `-z` to apply gzip compressions in order to reduce the file size
-  * Option `-v` to be verbose and display the progress of the operation
-  * Option `-f` to specify the path of the file archive
-* call `tar` to display the contents of the backup in file `$DIR/backup.tar` using
-  * Option `-t` to list the archive contents
-  * Option `-f` to specify the path of the file archive to list
+The following shell commands will
+* Run a Docker container
+* Mount your volume
+* Copy the contents of your volume to directory `$DIR`
+* Remove the container
 
 ```shell
-docker run --rm \
-  --volume ${VOLUME}:/volume \
-  --volume ${DIR}:/dest \
-  ubuntu \
-  tar -C /volume -czvf /dest/backup.tgz .
-tar tf ${DIR}/backup.tgz
+C=$(docker run --detach -v ${VOLUME}:/notebooks ubuntu sleep infinity)
```

Comment:
the separate container only makes sense, if we shut down the original one, which is probably a good idea.

Reply:
will be added in next push